client/src/components/ResultRow.js

```diff
@@ -31,9 +31,9 @@ class ResultRow extends Component {
     let arrow = '>';
     let originalPosting = '';
     let applyButton = '';
-    let { location } = this.props.result;
+    let location = this.props.result.city + ", " + this.props.result.state + " " + this.props.result.country;
```

Comment:
could you use a Javascript string template for this instead? They're a bit cleaner to read.